File: decompiled/4.0.2/be/ehealth/businessconnector/mycarenet/memberdata/service/MemberDataService.java
package be.ehealth.businessconnector.mycarenet.memberdata.service;

import be.ehealth.technicalconnector.exception.TechnicalConnectorException;
import be.ehealth.technicalconnector.service.sts.security.SAMLToken;
import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationRequest;
import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationResponse;

public interface MemberDataService {
   MemberDataConsultationResponse consultMemberData(SAMLToken var1, MemberDataConsultationRequest var2) throws TechnicalConnectorException;
}

File: src/main/kotlin/org/taktik/connector/business/memberdatav2/service/MemberDataService.kt
package org.taktik.connector.business.memberdatav2.service

import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationRequest
import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationResponse
import org.taktik.connector.technical.exception.TechnicalConnectorException
import org.taktik.connector.technical.service.sts.security.SAMLToken

interface MemberDataService {
    @Throws(TechnicalConnectorException::class)
    fun consultMemberData(token: SAMLToken, request: MemberDataConsultationRequest): MemberDataConsultationResponse
}
